commit 452e2b181626ca629970e6334af5692484df1668
Author: Fernando Mendoza <fermendozarcs@gmail.com>
Date:   Tue Mar 19 15:42:29 2019 +0100

    [stable/sonatype-nexus] Fix #11193 (#11833)
    
    * patch #11193
    
    Signed-off-by: Fernando Mendoza <fermendozarcs@gmail.com>
    
    * chart version bump
    
    Signed-off-by: Fernando Mendoza <fermendozarcs@gmail.com>
    
    * cosmetic refactor
    
    Signed-off-by: Fernando Mendoza <fermendozarcs@gmail.com>

diff --git a/stable/sonatype-nexus/Chart.yaml b/stable/sonatype-nexus/Chart.yaml
index 362efd9ad..c112f92fb 100644
--- a/stable/sonatype-nexus/Chart.yaml
+++ b/stable/sonatype-nexus/Chart.yaml
@@ -1,5 +1,5 @@
 name: sonatype-nexus
-version: 1.16.2
+version: 1.16.3
 appVersion: 3.15.2-01
 description: Sonatype Nexus is an open source repository manager
 keywords:
diff --git a/stable/sonatype-nexus/templates/ingress.yaml b/stable/sonatype-nexus/templates/ingress.yaml
index 3ef64aaa5..c1212dee2 100644
--- a/stable/sonatype-nexus/templates/ingress.yaml
+++ b/stable/sonatype-nexus/templates/ingress.yaml
@@ -11,6 +11,7 @@ metadata:
     {{- end }}
 spec:
   rules:
+  {{- if .Values.nexusProxy.env.nexusHttpHost }}
     - host: {{ .Values.nexusProxy.env.nexusHttpHost }}
       http:
         paths:
@@ -18,6 +19,8 @@ spec:
               serviceName: {{ template "nexus.fullname" . }}
               servicePort: {{ .Values.nexusProxy.port }}
             path: {{ .Values.ingress.path }}
+  {{- end }}
+  {{- if .Values.nexusProxy.env.nexusDockerHost }}
     - host: {{ .Values.nexusProxy.env.nexusDockerHost }}
       http:
         paths:
@@ -25,11 +28,16 @@ spec:
               serviceName: {{ template "nexus.fullname" . }}
               servicePort: {{ .Values.nexusProxy.port }}
             path: {{ .Values.ingress.path }}
+  {{- end }}
 {{- if .Values.ingress.tls.enabled }}
   tls:
     - hosts:
+      {{- if .Values.nexusProxy.env.nexusHttpHost }}
         - {{ .Values.nexusProxy.env.nexusHttpHost }}
+      {{- end }}
+      {{- if .Values.nexusProxy.env.nexusDockerHost }}
         - {{ .Values.nexusProxy.env.nexusDockerHost }}
+      {{- end }}
       {{- if .Values.ingress.tls.secretName }}
       secretName: {{ .Values.ingress.tls.secretName | quote }}
       {{- end }}
